packages/legend-graph/src/models/protocols/pure/v1/model/packageableElements/store/relational/mapping/V1_RelationalInputData.ts

```diff
@@ -18,18 +18,19 @@ import { hashArray } from '@finos/legend-shared';
 import type { Hashable } from '@finos/legend-shared';
 import { CORE_HASH_STRUCTURE } from '../../../../../../../../../MetaModelConst';
 import { V1_InputData } from '../../../../../model/packageableElements/mapping/V1_InputData';
+import type { V1_TestDataSource } from '../../modelToModel/mapping/V1_TestDataSource';
 
 export class V1_RelationalInputData extends V1_InputData implements Hashable {
   database!: string;
-  data!: string;
```

Comment:
We should still be able to read this property field, i.e be backward compatible. 
So we should read this property but during serialization we write this as 
`V1_StringTestDataSource` if the data property has been filled. 
The key is in the builder if the data property is present we build the relational input data that when transforming will give us V1_StringTestDataSource back. 
This should be for all InputData that has the data input.